client/src/pages/Actions.ts

```diff
@@ -1,7 +1,38 @@
-import { STORE_API, USER_API } from "src/constants";
+import {
+  STORE_API,
+  STORE_LIST_API,
+  USER_API,
+  ITEM_API,
+  ITEM_LIST_API,
+  DEFAULT_STORE_RADIUS,
+  PLACES_RADIUS_METERS,
+  PLACES_TYPES,
+  GOOGLE_MAP_PLACEHOLDER_ID,
+  GOOGLE_PLACES_SCRIPT_ID,
+} from "src/constants";
 import { fetchJson, extractIdentityToken } from "src/utils";
-import { isWeb, microapps } from "src/config";
-import { IdentityToken } from "src/interfaces";
+import { isWeb, google, microapps } from "src/config";
+import { IdentityToken, GeoLocation, emptyGeoLocation } from "src/interfaces";
+
+// Load up Google Maps Places API Service
+let map: any;
+let service: any;
```

Comment:
avoid global variables like the above

Reply:
👍 does js not have file/module-level variable scoping? o.O
My thought here is to initialize these once since there are potentially multiple functions in `Actions.ts` that would require us to interact with these objects.